### ISSUE TITLE
Looks like this addon cant be nested.

### ISSUE THREAD

**Issue by joedaniels29** [aexmachina/ember-cli-sass#96]
Not sure how to debug it exactly, but Im not able to get my apps styles folder, which is full of sass files, to compile when i have ember-cli-sass included from an addon. 

in most addons i've been able to mod them like so:
```javascript
included: function(app, parentAddon) {
     // see: https://github.com/ember-cli/ember-cli/issues/3718
    if (typeof app.import !== 'function' && app.app) {
      this.app = app = app.app;
    }
```

any ideas?

**Comment by aexmachina** [comment]
Possible duplicate of #63?

**Comment by aexmachina** [comment]
Fixed by #106
